templates/taxi/driver_list.html

```diff
@@ -8,6 +7,9 @@ <h1>Driver list</h1>
         <li>
           <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
```

Comment:
Ensure that the URL name 'taxi:driver-detail' is correctly defined in your URL configuration. If this view is not implemented yet, you should create it to avoid runtime errors.